panic: runtime error: index out of range [0] with length 0

full log:

```
WARN [11-18|02:17:08.504] The flag --pprofaddr is deprecated and will be removed in the future, please use --pprof.addr 
INFO [11-18|02:17:08.504] Starting pprof server                    addr=http://0.0.0.0:6060/debug/pprof
INFO [11-18|02:17:08.504] Starting Geth on Ethereum mainnet... 
WARN [11-18|02:17:08.504] Sanitizing cache to Go's GC limits       provided=16384 updated=10721
INFO [11-18|02:17:08.504] Enabling metrics collection 
INFO [11-18|02:17:08.508] Enabling metrics export to InfluxDB 
INFO [11-18|02:17:08.512] Maximum peer count                       ETH=5 LES=0 total=5
WARN [11-18|02:17:08.512] The flag --rpccorsdomain is deprecated and will be removed in the future, please use --http.corsdomain 
WARN [11-18|02:17:08.512] The flag --rpcapi is deprecated and will be removed in the future, please use --http.api 
INFO [11-18|02:17:08.512] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
INFO [11-18|02:17:08.518] Set global gas cap                       cap=25000000
INFO [11-18|02:17:08.519] Allocated trie memory caches             clean=7.33GiB dirty=0.00B
INFO [11-18|02:17:08.519] Allocated cache and file handles         database=/home/archive/geth_data/geth/chaindata cache=5.23GiB handles=524288
INFO [11-18|02:18:22.984] Opened ancient database                  database=/home/archive/geth_data/geth/chaindata/ancient
INFO [11-18|02:18:27.921] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Byzantium: 4370000 Constantinople: 7280000 Petersburg: 7280000 Istanbul: 9069000, Muir Glacier: 9200000, YOLO v2: <nil>, Engine: ethash}"
INFO [11-18|02:18:27.921] Disk storage enabled for ethash caches   dir=/home/archive/geth_data/geth/ethash count=3
INFO [11-18|02:18:27.921] Disk storage enabled for ethash DAGs     dir=/home/archive/.ethash               count=2
INFO [11-18|02:18:30.610] Initialising Ethereum protocol           versions="[65 64 63]" network=1 dbversion=8
INFO [11-18|02:19:31.608] Loaded most recent local header          number=11275361 hash="f09700…c3869b" td=18767999359520045058012 age=14h9m33s
INFO [11-18|02:19:31.609] Loaded most recent local full block      number=11275361 hash="f09700…c3869b" td=18767999359520045058012 age=14h9m33s
INFO [11-18|02:19:31.609] Loaded most recent local fast block      number=11275361 hash="f09700…c3869b" td=18767999359520045058012 age=14h9m33s
INFO [11-18|02:19:34.924] Loaded local transaction journal         transactions=0 dropped=0
INFO [11-18|02:19:34.925] Regenerated local transaction journal    transactions=0 accounts=0
INFO [11-18|02:19:35.752] Starting peer-to-peer node               instance=Geth/v1.9.24-stable-cc05b050/linux-amd64/go1.15.5
INFO [11-18|02:19:35.955] New local node record                    seq=44 id=4ca81c2b43e24e8a ip=127.0.0.1 udp=30303 tcp=30303
INFO [11-18|02:19:35.959] Started P2P networking                   self=enode://a722b2fe56cfdadaf67ca5cb6ddb0ac66bc743d04409023577a143dd1a505cd1457e62b7636afcac8c790aade8c29cc114965d0d4ee67e5440f41f84c4719662@127.0.0.1:30303
INFO [11-18|02:19:35.960] IPC endpoint opened                      url=/home/archive/geth_data/geth.ipc
INFO [11-18|02:19:35.961] HTTP server started                      endpoint=10.10.42.123:8545 cors=* vhosts=*
INFO [11-18|02:19:39.164] New local node record                    seq=45 id=4ca81c2b43e24e8a ip=178.19.221.38 udp=30303 tcp=30303
INFO [11-18|02:19:39.823] Mapped network port                      proto=tcp extport=30303 intport=30303 interface="UPNP IGDv1-IP1"
INFO [11-18|02:19:41.180] Mapped network port                      proto=udp extport=30303 intport=30303 interface="UPNP IGDv1-IP1"
INFO [11-18|02:19:45.979] Looking for peers                        peercount=4 tried=13 static=0
INFO [11-18|02:19:46.795] Block synchronisation started 
INFO [11-18|02:19:55.980] Looking for peers                        peercount=4 tried=23 static=0
INFO [11-18|02:20:10.959] Looking for peers                        peercount=4 tried=22 static=0
INFO [11-18|02:20:26.536] Looking for peers                        peercount=4 tried=11 static=0
INFO [11-18|02:20:37.046] Looking for peers                        peercount=4 tried=7  static=0
INFO [11-18|02:20:47.243] Looking for peers                        peercount=4 tried=12 static=0
INFO [11-18|02:20:57.593] Downloader queue stats                   receiptTasks=0 blockTasks=3009 itemSize=18.54KiB throttle=3535
INFO [11-18|02:20:59.256] Looking for peers                        peercount=4 tried=2  static=0
WARN [11-18|02:21:21.583] Checkpoint challenge timed out, dropping id=e92b653b1053099d conn=dyndial addr=47.94.111.100:30303             type=Parity-Ethereum/v2.5...
INFO [11-18|02:21:22.244] Looking for peers                        peercount=4 tried=6  static=0
WARN [11-18|02:21:33.982] Served eth_getTransactionReceipt         conn=10.10.42.2:36442 reqid=9958 t="64.036µs" err="invalid argument 0: hex string has length 40, want 64 for common.Hash"
INFO [11-18|02:21:34.256] Looking for peers                        peercount=4 tried=25 static=0
WARN [11-18|02:21:38.628] Served eth_getTransactionReceipt         conn=10.10.42.2:36442 reqid=10009 t="86.035µs" err="invalid argument 0: hex string has length 40, want 64 for common.Hash"
INFO [11-18|02:21:44.319] Looking for peers                        peercount=4 tried=12 static=0
WARN [11-18|02:22:08.011] Served eth_getTransactionReceipt         conn=10.10.42.2:40288 reqid=35632 t="104.033µs" err="invalid argument 0: json: cannot unmarshal hex string of odd length into Go value of type common.Hash"
WARN [11-18|02:22:36.998] Served eth_getTransactionReceipt         conn=10.10.42.2:44044 reqid=10419 t="74.155µs"  err="invalid argument 0: hex string has length 40, want 64 for common.Hash"
WARN [11-18|02:31:05.094] Served debug_traceTransaction            conn=10.10.42.2:52444 reqid=78106 t=1.015367942s err="transaction 2af2e4d514b74392b6b032a71fd16b443192c449bd760a07714a5dc14fa937b9 not found"
WARN [11-18|02:31:16.872] Served debug_traceTransaction            conn=10.10.42.2:53452 reqid=78455 t=852.634222ms err="transaction 6ce67b8b25a9b1f8349b37c0d32c426432c935c8bed112680153ac898563c220 not found"
WARN [11-18|02:31:20.532] Served debug_traceTransaction            conn=10.10.42.2:53452 reqid=78522 t=944.768697ms err="transaction 7e835262615d653619dd3239a3da8f324bc4216d59c6819e71e4da76442e68bb not found"
panic: runtime error: index out of range [0] with length 0

goroutine 10 [running]:
github.com/ethereum/go-ethereum/metrics.ReadCPUStats(0xc000272520)
	github.com/ethereum/go-ethereum/metrics/cpu_enabled.go:35 +0x1a6
github.com/ethereum/go-ethereum/metrics.CollectProcessMetrics(0xb2d05e00)
	github.com/ethereum/go-ethereum/metrics/metrics.go:101 +0x9d2
created by main.prepare
	github.com/ethereum/go-ethereum/cmd/geth/main.go:342 +0x398

```

FWIW: happened after the archive node db repair - so *might* be due to this - but as the stacktrace does not really sound related dropping this here

--- a/metrics/cpu_enabled.go
+++ b/metrics/cpu_enabled.go
@@ -31,6 +31,10 @@ func ReadCPUStats(stats *CPUStats) {
 		log.Error("Could not read cpu stats", "err", err)
 		return
 	}
+	if len(timeStats) == 0 {
+		log.Error("Empty cpu stats")
+		return
+	}
 	// requesting all cpu times will always return an array with only one time stats entry
 	timeStat := timeStats[0]
 	stats.GlobalTime = int64((timeStat.User + timeStat.Nice + timeStat.System) * cpu.ClocksPerSec)

